WatchFaceAlphaKotlin/app/src/main/AndroidManifest.xml

```diff
@@ -49,25 +49,25 @@
             android:permission="android.permission.BIND_WALLPAPER">
             <intent-filter>
                 <action android:name="android.service.wallpaper.WallpaperService" />
-
                 <category android:name="com.google.android.wearable.watchface.category.WATCH_FACE" />
             </intent-filter>
 
-            <meta-data
-                android:name="com.google.android.wearable.standalone"
-                android:value="true" />
```

Comment:
Is this tag no longer required/included automatically?

Reply:
It still exists at the top level, somehow one ended up in the service itself.